recipes/requests-negotiate-sspi/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "requests-negotiate-sspi" %}
+{% set version = "0.5.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/brandond/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 1566084baec815d11f2df07f01e078343a4bbbbb6e5ba163cc9e1b5f7d751f0f
+
+build:
+  skip: True  # [not win]
```

Comment:
Can this be noarch? It’s okay if it’s missing dependencies on windows

Reply:
IIUC it's a Windows only package. As it is, it's implemented via `pywin32` so is currently Windows only.